doc/atcommands.txt

```diff
@@ -780,25 +780,9 @@ To drop all weapons in inventory...
 
 ---------------------------------------
 
-@stockall {<item type>}
+@stockall
```

Comment:
Sorry, maybe I didn't explain myself well enough before. I meant that you can still stock based on different item types. I went back and looked at the item_db TradeRestrictions, and there is indeed a `NoCart` flag, which means I was wrong from the start about the counting part!

We can revert this commit and then just add a message at the end of the command like `@dropall` about how many were moved and how many weren't.